### PR TITLE
Move Jenkins scripts to root of System Drive

### LINKED ISSUE #79: Align Jenkins configuration to be in-line with QDE Appliance
In the appliance, scripts that Jenkins jobs rely on live on the root of C:\ in a scripts folder. 

To provide parity with the appliance experience, we need to adjust the way we handle the configuration of Jenkins during that step of the Guide.

### PR DESCRIPTION
**Pull request by steviecoaster**
This PR will fix #79 by moving the Jenkins scripts to the root of the System Drive into  a `scripts` folder. 

This brings parity with QDE appliance.

### CODE REVIEW

**Comment by steviecoaster** [comment]
Remove from DRAFT once tested and approved

File: Start-C4bJenkinsSetup.ps1 [hunk 1]
```diff
@@ -34,6 +34,14 @@ choco pin add --name="'jenkins'" --version="'2.222.4'" --reason="'Next version i
 Write-Host "Giving Jenkins 30 seconds to complete background setup..." -ForegroundColor Green
 Start-Sleep -Seconds 30  # Jenkins needs a moment
 
+# Long winded way to get the scripts for Jenkins jobs into the right place, but easier to mainntain going forward
+$root = Split-Path -Parent $MyInvocation.MyCommand.Definition
+$systemRoot = $env:SystemDrive
```
Comment (adilio) [inline Start-C4bJenkinsSetup.ps1:39]:
```suggestion
$systemRoot = $env:SystemDrive + '\'
```

**Review by adilio — CHANGES_REQUESTED** [review]
Once the trailing slash suggestion is fixed, we're good to go.

You'll need to rebase first, but after that the code is tested and good to go.

**Review by ryanrichter94 — APPROVED** [review]
LGTM 🚀

**Review by adilio — APPROVED** [review]
LGTM, and tested! 🚀

**Pull request merged**